src/coins/coins.ts

```diff
@@ -490,6 +490,20 @@ export const basicCoins: BasicCoin[] = [
       },
     },
   },
+  // > BTCN
+  {
+    key: 'BTCN' as CoinKey,
```

Comment:
I would suggest to add BTCN as a CoinKey in types package. We may use it in our BE in the future. Just my experience ;D

Reply:
Personally I think the `CoinKey` should be deprecated. Do you have an example why we need it in the BE? `findDefaultToken` will be deleted once we get rid of `data-types` since it's very unsafe. The method just throws an error if a token doesn't exist on a chain, which can always happen. For that we have the token service. 
So I don't really see another usecase for `CoinKey`. 90% of the tokens we support don't have a `CoinKey` defined anyway right? Everything that's not coming from our data-types.

Reply:
But i'm open for discussion of course!